crates/modelardb_compression/src/lib.rs

```diff
@@ -309,60 +384,18 @@ impl<'a> CompressedSegmentBuilder<'a> {
 
         self.swing_could_fit_all =
             self.swing_could_fit_all && self.swing.fit_data_point(timestamp, value);
-
-        // Gorilla uses lossless compression and cannot exceed the error bound.
-        if self.gorilla.length < GORILLA_MAXIMUM_LENGTH {
-            self.gorilla.compress_value(value);
-        }
     }
 
     /// Return [`true`] if any of the current models can represent additional
     /// values, otherwise [`false`].
     fn can_fit_more(&self) -> bool {
-        self.pmc_mean_could_fit_all
-            || self.swing_could_fit_all
-            || self.gorilla.length < GORILLA_MAXIMUM_LENGTH
+        self.pmc_mean_could_fit_all || self.swing_could_fit_all
     }
 
-    /// Store the model that requires the smallest number of bits per value in
-    /// `compressed_record_batch_builder`. Returns the index of the first value
-    /// in `uncompressed_values` the selected model could not represent.
-    fn finish(self, compressed_record_batch_builder: &mut CompressedSegmentBatchBuilder) -> usize {
+    /// Return the model that requires the smallest number of bits per value if used for a segment.
```

Comment:
Maybe change to `the fewest number of bytes per value` since that unit is used elsewhere. I also don't fully understand the meaning of `if used for a segment` here.

Reply:
Fixed and removed the comment inside the method as it just repeated the doc comment.